src/rno-g-acq.c

```diff
@@ -578,20 +578,18 @@ int flower_initial_setup()
 }
 
 
+static int record_timimg()
+{
+  printf("Performing timing measurements. This should just take a few minutes.");
+  system("python3 /home/rno-g/stationrc/record_timings.py -n 5 --data_dir /power/timing");
```

Comment:
Is `/power/timing` guaranteed to exist?  Maybe should make it part of the `install` target of the Makefile to make sure it exists. 

It also may make sense eventually (not necessarily now...) to make this command a configurable string in the config file?

Reply:
Well I made sure it exists on the test station :)

Reply:
is it supposed to be /data/timing ? :)

Reply:
autsch